docs/installation.md

```diff
@@ -58,10 +59,18 @@ services:
 
 ```
 
+The MA team will support docker installs that are installed per the above instructions. For clarity, to receive support from the MA team:
+- The docker install must be a simple standalone container (e.g. not using kubernetes)
+- MA, HA and all players must be on the same flat network with no VLANs 
+- Host networking or macvlan is a requirement for the docker container
```

Comment:
```suggestion
- Music Assistant needs direct (layer 2) acces sto the network to properly discover and stream to players. So either host networking or macvlan networking is a hard requirement for the docker container.
- The docker container runs on a modern operating system with the docker engine. NAS operating systems like Synology, UInRaid etc. may work but you may run into trouble. Better run HA+MA as a full HAOS Virtual machine in that case
```

Reply:
maybe point at the HA documentation for running the container, we follow those.

Reply:
Small mistype above line 65 'acce sto'

Reply:
fixed on both